<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>labelToolPage</class>
 <widget class="QWidget" name="labelToolPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>507</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>507</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="txtDataset">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>441</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtOutput">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>441</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbReturnMainPage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSetDataset">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Dataset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSetOutput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Output</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txtOutputProcess">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>541</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbExecute">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Execute</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listAddedMasks">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>121</y>
     <width>541</width>
     <height>121</height>
    </rect>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::InternalMove</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAddMask">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>AddMask</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDeleteMask">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>DeleteMask</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
